src/elasticodbc/es_types.h

```diff
@@ -295,7 +295,6 @@ typedef struct ColumnInfo {
     }
 } ColumnInfo;
 
-// TODO: AE-111 determine if there's a better way to hold the server_info -
 // perhaps a vector of smart pointers?
```

Comment:
Should be removed as part of previous line

Reply:
removed